### ISSUE TITLE
Doesn't install on Node 0.10.

### ISSUE THREAD

**Issue by NodeGuy** [eschulte/node-gpg#2]
The installer uses `node-waf` which was deprecated in Node 0.8.  It should be using [node-gyp](https://github.com/TooTallNate/node-gyp) instead.

**Comment by biebg** [comment]
have you fixed this problem? i have tried node-gyp ,and build success, but still have error  "gpg_module not found."

**Comment by eschulte** [comment]
No.  I haven't touched this repository in a couple of years, and won't have time to provide any sort of maintenance in the forseable future.
